src/panels/developer-tools/template/developer-tools-template.ts

```diff
@@ -181,17 +181,21 @@ class HaPanelDevTemplate extends LitElement {
                 >`
               : nothing}
             ${this._templateResult
-              ? html`${this.hass.localize(
-                    "ui.panel.developer-tools.tabs.templates.result_type"
-                  )}:
-                  ${resultType}
-                  <!-- prettier-ignore -->
-                  <pre class="rendered ${classMap({
-                    [resultType]: resultType,
-                  })}"
-                  >${type === "object"
-                    ? JSON.stringify(this._templateResult.result, null, 2)
-                    : this._templateResult.result}</pre>
+              ? html`<pre
+                    class="rendered ${classMap({
+                      [resultType]: resultType,
+                    })}"
+                  >
+                    ${type === "object"
+                      ? JSON.stringify(this._templateResult.result, null, 2)
+                      : this._templateResult.result}</pre
+                  >
```

Comment:
```suggestion
                  >${type === "object"
                      ? JSON.stringify(this._templateResult.result, null, 2)
                      : this._templateResult.result}</pre
                  >
```
Have to be careful with whitespaces in `pre` tags because they get rendered